internal/ark/sync/backups_sync_svc.go

```diff
@@ -70,8 +72,19 @@ func (s *BackupsSyncService) SyncBackups(clusterManager *cluster.Manager) error
 	// iterate through clusters and sync backups for each of them
 	for _, cluster := range clusters {
 		log := s.logger.WithField("clusterID", cluster.GetID())
+
+		status, err := cluster.GetStatus()
+		if err != nil {
+			log.Error(emperror.Wrap(err, "could not get cluster status"))
```

Comment:
Use an error handler here

Reply:
Thanks @sagikazarmark , I'll fix this in an upcoming PR!